lib_standard_app/swap_lib_calls.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+/* This file is the shared API between Exchange and the apps started in Library mode for Exchange
+ *
+ * DO NOT MODIFY THIS FILE IN APPLICATIONS OTHER THAN EXCHANGE
+ * On modification in Exchange, forward the changes to all applications supporting Exchange
```

Comment:
To update

Reply:
This is a pure copy from exchange app, Isn't it what we need?